v2/encoding_test.go

```diff
@@ -25,6 +26,54 @@ func TestDecodeText(t *testing.T) {
 	}
 }
 
+func TestDecodeTextParallel(t *testing.T) {
```

Comment:
Please add a link to the issue:
```suggestion
// See https://github.com/bogem/id3v2/pull/74
func TestDecodeTextParallel(t *testing.T) {
```